src/storage.cpp

```diff
@@ -225,18 +221,15 @@ namespace libtorrent
 
 		int file_op(int const file_index
 			, std::int64_t const file_offset
-			, int const size
-			, file::iovec_t const* bufs, storage_error& ec)
+			, span<file::iovec_t const> bufs, storage_error& ec)
 			final
 		{
 			if (m_storage.files().pad_file_at(file_index))
 			{
 				// writing to a pad-file is a no-op
-				return size;
+				return bufs_size(bufs);
```

Comment:
this function used to take the size, as pre-computed. now it doesn't and have to compute it itself